### PR TITLE
[Merged by Bors] - Added image_pull_secrets to PodBuilder

### PR DESCRIPTION
**Pull request by razvan**

### CODE REVIEW

File: src/builder/pod/mod.rs [hunk 1]
```diff
@@ -5,6 +5,7 @@ pub mod volume;
 use crate::builder::meta::ObjectMetaBuilder;
 use crate::error::{Error, OperatorResult};
 
+use k8s_openapi::api::core::v1::LocalObjectReference;
 use k8s_openapi::{
     api::core::v1::{
         Affinity, Container, NodeAffinity, NodeSelector, NodeSelectorRequirement, NodeSelectorTerm,
```
Comment (maltesander) [inline src/builder/pod/mod.rs:11]:
```suggestion
use k8s_openapi::{
    api::core::v1::{
        Affinity, Container, LocalObjectReference, NodeAffinity, NodeSelector, NodeSelectorRequirement, NodeSelectorTerm,
```

**Review by maltesander — CHANGES_REQUESTED** [review]
One minor thing

File: src/builder/pod/mod.rs (hunk 1 above)
Comment (maltesander) [reply src/builder/pod/mod.rs]: @razvan Sorry that wasnt properly fmt formatted...

**Review by maltesander — APPROVED** [review]
LGTM.

**Comment by razvan** [comment]
bors merge

**Pull request closed without merging**